engine/render/backend/api/BackendDX.cpp

```diff
@@ -0,0 +1,14 @@
+#include "BackendDX.h"
+
+#include <cstddef>
+#include <cstring>
+#include <map>
+#include <stack>
+#include <string>
+
+#include <core/Assertions.h>
+#include <core/Logger.h>
+
+using namespace chira;
+
+CHIRA_CREATE_LOG(DX);
```

Comment:
DX11 and DX12 are fundamentally different APIs, make this specifically a DX11 backend -- you won't be able to share much between 11 and 12 to my knowledge

Reply:
:+1: good to know. I hadn't touched the DX12 docs yet. only DX11.